core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -96,13 +97,17 @@ public synchronized static DockerClientFactory instance() {
 
     /**
      *
-     * @return a new initialized Docker client
+     * @return an initialized Docker client
```

Comment:
Since the factory is already singelton, does this mean, that the client is a singleton now as well?

Reply:
it was a singleton for years, actually :)